### DISCUSSION TITLE
Expand transmog limitations

### DISCUSSION THREAD

**Discussion by jdmccathie** [chromiecraft/chromiecraft#5752] [Suggestions]
For many classes their best in slot gear will usually force them to wear a combination of plate, mail, letter, or cloth gear. This makes it tricky to get a consistent looking transmog when armour types are restricted to their own armour class.
Please remove the restrictions so armour can be transmogged to an armour which the player can wear.

**Comment by 55Honey** [comment]
Hi @jdmccathie 
This is a duplicate. See https://github.com/chromiecraft/chromiecraft/discussions?discussions_q=is%3Aopen+transmog
If everyone opens a new thread for a subject, there will hardly be a discussion.

Regards
